package.json

```diff
@@ -7,7 +7,8 @@
   "dependencies": {
     "prelude-extension": "^0.1.0",
     "prelude-ls": "^1.1.1",
-    "tether": "^1.1.1"
+    "tether": "^1.1.1",
+    "uuid": "^8.3.2"
```

Comment:
can uuid be a peer dependency?

Reply:
I think we can, but then we'd need to add it to our internal codebase.  it didn't make sense to me to assume that the project would have uuid (kind of like why 'react' is a peer dependency -- it wouldn't make sense to use react-selectize2 outside of the context of a react app)

Reply:
updating this.  I just missed it in our node_modules already